huaweicloud/services/acceptance/dds/resource_huaweicloud_dds_lts_log_test.go

```diff
@@ -0,0 +1,122 @@
+package dds
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk/openstack/dds/v3/instances"
+	"github.com/chnsz/golangsdk/pagination"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/dds"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getDdsLtsLogResourceFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	var (
+		getDdsLtsLogHttpUrl = "v3/{project_id}/instances/logs/lts-configs"
+		getDdsLtsLogProduct = "dds"
+	)
+
+	getDdsLtsLogClient, err := cfg.NewServiceClient(getDdsLtsLogProduct, acceptance.HW_REGION_NAME)
+	if err != nil {
+		return nil, fmt.Errorf("error creating DDS Client: %s", err)
+	}
+
+	getDdsLtsLogPath := getDdsLtsLogClient.Endpoint + getDdsLtsLogHttpUrl
+	getDdsLtsLogPath = strings.ReplaceAll(getDdsLtsLogPath, "{project_id}", getDdsLtsLogClient.ProjectID)
+
+	getDdsLtsLogResp, err := pagination.ListAllItems(
+		getDdsLtsLogClient,
+		"offset",
+		getDdsLtsLogPath,
+		&pagination.QueryOpts{MarkerField: ""})
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving DDS LTS configs: %s", err)
+	}
+
+	getDdsLtsLogRespJson, err := json.Marshal(getDdsLtsLogResp)
+	if err != nil {
+		return nil, fmt.Errorf("error marshaling DDS LTS configs: %s", err)
+	}
+
+	var getDdsLtsLogRespBody interface{}
+	err = json.Unmarshal(getDdsLtsLogRespJson, &getDdsLtsLogRespBody)
+	if err != nil {
+		return nil, fmt.Errorf("error unmarshaling DDS LTS configs: %s", err)
+	}
+
+	instanceLtsConfig := dds.FilterInstanceLtsConfig(state.Primary.ID, getDdsLtsLogRespBody)
+	ltsConfigs := utils.PathSearch("lts_configs", instanceLtsConfig, make([]interface{}, 0)).([]interface{})
```

Comment:
```suggestion
	ltsConfigs ：= utils.PathSearch(fmt.Sprintf("instance_lts_configs[?instance.id=='%s']|[0].lts_configs|[0]"), getDdsLtsLogRespBody,nil)
```
and fix other places

Reply:
done